bindings/protocols/coap/index.html

```diff
@@ -142,23 +147,57 @@ <h3>Content Negotiation</h3>
                 The CoAP Accept option is used by clients to request a particular content format, while the <code>Content-Format</code> option is used by clients and servers to indicate the content format of the representation in requests and responses, respectively.
             </p>
             <p>
-                The ... can be used to provide a list of one or more content formats that the Thing is able to provide in a CoAP response for the operation.
-                The Consumer selects the most suitable one and sets the <code>Accept</code> option in the request accordingly.
+                Mapping the string-based <code>contentType</code> and <code>contentCoding</code> fields to a numeric Content-Format can be difficult for a Consumer for a number of reasons:
+                First, a Consumer needs to be aware of all potential mappings between the string-based and the numerical representations.
+                Furthermore, since there is no well-defined algorithm for converting a combination of <code>contentType</code> and <code>contentCoding</code> to a CoAP Content-Format, the process might be prone to error.
             </p>
-            <pre id="example-request-accept" class="example" title="...">
-{
-    ...
-}
-</pre>
             <p>
-                The ... can be used to provide a list of one or more content formats that the Thing is willing to accept in a CoAP request for the operation.
-                The Consumer selects the most suitable one and sets the <code>Content-Format</code> option in the request accordingly.
+                Therefore, this document introduces the vocabulary term <code>cov:contentFormat</code> for providing a hint to simplify the conversion for a Consumer.
+                It can be used in all Thing Description classes that contain the <code>contentType</code> and <code>contentCoding</code> fields.
+                If the field <code>cov:contentFormat</code> is present, it must match the string representation expressed by <code>contentType</code> and <code>contentCoding</code>.
+                Corresponding with the default <code>contentType</code> value <code>application/json</code>, Consumers are supposed to assume a Content-Format value 50 if <code>cov:contentFormat</code> and <code>application/json</code> is not present.
             </p>
-            <pre id="example-request-format" class="example" title="...">
-{
-    ...
-}
-</pre>
+            <p>
+                Consumers may use the <code>cov:contentFormat</code> in a Form's <code>response</code> member object to derive the value for an Accept option in their request.
+                However, TDs may also contain a <code>cov:accept</code> to explicitly require a Consumer to provide an Accept option.
```

Comment:
@ektrah @egekorkan I guess we can simply remove the sentence:

```suggestion
                Forms may also contain a <code>cov:accept</code> field to explicitly require a Consumer to provide an Accept option.
```

However, I think one concern I still had was that the `cov:accept` field leads to some redundancy, since its value should probably always be derivable from either the form or the response member. Therefore, I am wondering if this field could also be a boolean, which would indicate that an Accept option must be used.

Then again, it is not that big of a deal, and we could also just this version in order to not prolong the discussion more than needed.